.github/workflows/build.yml

```diff
@@ -85,6 +85,26 @@ jobs:
       run: script/test build check
     - name: Codecov
       uses: codecov/codecov-action@v1
+    
+  rest-check:
```

Comment:
Seems otbr-web does not have an independent github action test, if otbr-rest replace this, does it need this remove/add test? @wgtdkp